finance-api/concepts/how-card-payments-work-at-Zettle.md

```diff
@@ -0,0 +1,21 @@
+How card payments work at Zettle
+===
+For every merchant who makes transactions with Zettle, they have an Zettle account that handles those transactions.
+
+As shown in [Figure 1 Zettle card transaction flow](#Zettle-card-transaction-flow), after a merchant makes a transaction with Zettle, the money of the transaction will go through the following phases in sequence:
+1. Recorded in the merchant's Zettle preliminary account:
+ 
+   Zettle checks the money with a third-party acquirer who confirms the transaction.
+   
+2. Recorded in the merchant's Zettle liquid account:
+
+   After the third-party acquirer confirms the transaction, the money is ready to be paid out to the merchant.
+   
+3. Recorded as paid-out: 
+  
+   Zettle reconciles the transaction to make sure that the correct amount of money will be paid out from the merchant's Zettle account to the merchant's bank account. Then the merchant will receive the money.
+   
+> **Note:** It usually takes two days for the money to be paid out. However, it may take longer, depending on the communication between third-party acquirers and banks.
+
+<figure ><img id="Zettle-card-transaction-flow" src="../images/Zettle-card-transaction-flow.svg" alt="This card transaction flow shows how Zettle handles transaction money." >  <figcaption><b>Figure 1. </b>Zettle card transaction flow</figcaption></figure>
```

Comment:
"The third acquirer" doesn't make sense.
And also why expose this piece of internal architecture to anyone outside of Zettle?
Why do we need to add this diagram?

Reply:
For us it would be interesting not to mention, but want to hear from @Astro-Choco .